build-tools/build-indexes

```diff
@@ -589,11 +571,7 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 				formatSlices[usedTier] = tiers.length;
 			}
 			if (!tierTable[tier]) continue;
```

Comment:
wdym

Reply:
actually now that I'm thinking about it that behavior might be intentional. you should definitely bring that line back to its original location though since it looks like you removed it